src/EditorFeatures/Test/EditAndContinue/EditAndContinueWorkspaceServiceTests.cs

```diff
@@ -4562,7 +4562,7 @@ public async Task UnitTestingHotReloadServiceTest()
             // Rude edit
             result = await hotReload.EmitSolutionUpdateAsync(solution, commitUpdates: true, CancellationToken.None);
             AssertEx.Equal(
-                new[] { "ENC0009: " + string.Format(FeaturesResources.Updating_the_type_of_0_requires_restarting_the_application, FeaturesResources.method) },
+                new[] { "ENC0021: " + string.Format(FeaturesResources.Adding_0_requires_restarting_the_application, FeaturesResources.type_parameter) },
```

Comment:
>ENC0021 [](http://example.com/codeflow?start=25&length=7)

Haha, as we keep removing rude edits it gets harder to test that we are reporting rude edits correctly :)

Reply:
Soon we'll need a special "You can't insert a comment that contains `__rude_edit__`" just so we stop chasing our tail.